### PR TITLE
WIP: Hotfix/sponsorship link

### PR DESCRIPTION
**Pull request by Hyperion101010**
### Description

It makes the default link color as light blue on sponsor ebook button.
Closes #

### Technical

A minor fix. The issue was caused due to the improper use of class priority in CSS classes. 

### Testing
Screenshot from 2019-08-02 01-31-58

1>First rebuild the docker files through docker-compose exec web bash followed by make css.
2>Then visit the Odyssey book link and see the default link color.

### CODE REVIEW

File: openlibrary/macros/databarAuthor.html [hunk 1]
```diff
@@ -27,7 +27,11 @@ <h3>Add to list</h3>
             <div class="floaterAdd" id="addList">
                 <div class="floaterHead">
                     <h2>Create a new list</h2>
+<<<<<<< HEAD
```
Comment (mekarpeles) [inline openlibrary/macros/databarAuthor.html]: I see a git conflict here

File: .eslintrc.json [hunk 2]
```diff
@@ -5,22 +5,22 @@
     "jquery": true
   },
   "parserOptions": {
-		"sourceType": "module",
-		"ecmaVersion": 6
-	},
-  "globals": {
+    "sourceType": "module",
```
Comment (mekarpeles) [inline .eslintrc.json]: it seems like maybe the spacing has changed on this and a lot of other files!

File: openlibrary/macros/LoanStatus.html [hunk 3]
```diff
@@ -80,11 +80,13 @@
...
   $ betterworldbooks_metadata = get_betterworldbooks_metadata(isbn)
-  $if user and user.is_sponsor() and isbn and all(page.get(x) for x in ['publishers', 'title', 'publish_date', 'covers']) and page.works and page.works[0].get('authors') and betterworldbooks_metadata and ('price_amt' in betterworldbooks_metadata) and (float(betterworldbooks_metadata['price_amt']) < 15.00):
+  $if user and user.is_sponsor() and isbn and all(page.get(x) for x in ['publishers', 'title', 'publish_date', 'covers']) and
+  page.works and page.works[0].get('authors') and betterworldbooks_metadata and ('price_amt' in betterworldbooks_metadata)
+  and (float(betterworldbooks_metadata['price_amt']) < 15.00):
     $ user_id = user.key.split("/")[-1] if user.key else ''
-    <a href="https://archive.org/donate?context=ol&userid=$(user_id)&campaign=pilot&type=book-sponsorship&isbn=$(isbn)" class="cta-btn cta-btn--sponsor">Sponsor eBook</a>
+    <a href="https://archive.org/donate?context=ol&userid=$(user_id)&campaign=pilot&type=book-sponsorship&isbn=$(isbn)" class="cta-btn cta-btn--sponsor cat-btn--sponsor:link">Sponsor eBook</a>
```
Comment (mekarpeles) [inline openlibrary/macros/LoanStatus.html]: I don't think we should need `cat-btn--sponsor:link` here -- also this is typo'd, even if we did need it :)

**Comment by tabshaikh** [comment]
@Hyperion101010 Thank you so much for your efforts and congrats on your first piece of code in open library repo :tada: I have cherry-picked your commit and added to my pr #2267 with you as the author :) .

**Pull request closed without merging**